include/service/entities/article_service.js

```diff
@@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
 };
 
 /**
+ * Updates articles
+ *
+ * @param {String} articleId	id of article
+ * @param {Object} options	
+ * @param {Object} fields	fields to update	
+ * @param {Function} cb      Callback function
+ */
+ArticleService.prototype.update = function(articleId, options, fields, cb) {
```

Comment:
k


On Mon Feb 02 2015 at 6:01:20 PM Brian <notifications@github.com> wrote:

> In include/service/entities/article_service.js
> <https://github.com/pencilblue/pencilblue/pull/454#discussion_r23943498>:
>
> > @@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
> >  };
> >
> >  /**
> > + * Updates articles
> > + *
> > + * @param {String} articleId	id of article
> > + * @param {Object} options	
> > + * @param {Object} fields	fields to update	
> > + * @param {Function} cb      Callback function
> > + */
> > +ArticleService.prototype.update = function(articleId, options, fields, cb) {
>
> You also need to verify that fields is an actual object since it is
> required. You can use the pb.utils.isObject(fields) function to do that
> check. If it isn't an object just return cb(new Error('The fields
> parameter is required'));
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pencilblue/pencilblue/pull/454/files#r23943498>.
>